urdf/husky_ur5/base_with_manipulator.urdf.xacro

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
```

Comment:
Instead, modify the urdf/description.xacro, something along the lines of

<xacro:arg name="ur5_enabled" default="false" />

<xacro:husky..

<xacro:if value="$(arg ur5_enabled)">
<xacro:ur5_robot..
<xacro:include filename="...husky_ur5_glue.xacro">